akd_mysql/src/mysql.rs

```diff
@@ -870,6 +870,13 @@ impl Storage for AsyncMySqlDatabase {
         }
 
         // cache miss, log a real sql read op
+        self.get_direct::<St>(id).await
```

Comment:
Ah that's my slip-up. ```get_direct``` should not alter the cache at all, simply returning the result, and then if ```get``` wants, it can cache the object. Callers of ```get_direct``` should not play with the cache in any way.